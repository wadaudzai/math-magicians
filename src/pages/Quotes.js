const Quotes = () => (
  <h1 className="quote">
    Mathematics isnt about numbers, equations,computations
    or algorithms. it is about understanding. - William Paul Thrutson
  </h1>
);

export default Quotes;
