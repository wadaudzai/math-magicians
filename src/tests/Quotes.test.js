import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../pages/Quotes';

describe('Check Quotes Component', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<Quotes />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
