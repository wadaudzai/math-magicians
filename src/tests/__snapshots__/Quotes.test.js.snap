// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check Quotes Component renders correctly 1`] = `
<h1
  className="quote"
>
  Mathematics isnt about numbers, equations,computations or algorithms. it is about understanding. - William Paul Thrutson
</h1>
`;
